<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.00568182 rgba(1, 142, 190, 255), stop:1 rgba(255, 255, 255, 255))
}
</string>
  </property>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:rgb(0, 255, 0);
font: 63 12pt &quot;Verdana Pro Semibold&quot;;
background-color:rgb(54, 54, 81);
border-radius:8px;
}
QPushButton:hover{
	background-color:rgb(41, 41, 61);
}
</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
	color:rgb(54, 54, 81);
	font: 63 14pt &quot;Verdana Pro Semibold&quot;;
border-radius:8px;
background-color:transparent;
border:1px solid  rgb(54, 54, 81);
padding:2px
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color:rgb(54, 54, 81);
	font: 63 14pt &quot;Verdana Pro Semibold&quot;;
border-radius:8px;
background-color:transparent;
border:1px solid  rgb(54, 54, 81);
padding:2px
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color:rgb(54, 54, 81);
	font: 63 14pt &quot;Verdana Pro Semibold&quot;;
border-radius:8px;
background-color:transparent;
border:1px solid  rgb(54, 54, 81);
padding:2px
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>content</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(54, 54, 81);
	font: 63 10pt &quot;Verdana Pro Semibold&quot;;
	background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>subject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#fff;
font: 63 10pt &quot;Verdana Pro Semibold&quot;;
background-color:rgb(255, 66, 66);
border-radius:8px;
}
QPushButton:hover{
	background-color:rgb(255, 20, 20);
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
